UefiPayloadPkg/SPI/lpc.h

```diff
@@ -0,0 +1,134 @@
+#include <Include/PiDxe.h>
```

Comment:
No need to have whole this file, put only necessary defines to the files where needed

Reply:
Removed it